python/download_tile.py

```diff
@@ -0,0 +1,77 @@
+from looker_sdk import client, models
+import time
+import sys
+
+sdk = client.setup("../looker.ini")
+
+def get_dashboard(title: str):
+  """Get a dashboard by title"""
+  dashboard = next(iter(sdk.search_dashboards(title=title)), None)
+  if not dashboard:
+    print(f"dashboard {title} was not found")
+  return dashboard
+
+def get_tile(dash: models.Dashboard, title: str):
+  """Find a dashboard tile by title"""
+  title = title.lower()
+  found = None
+  assert dash.dashboard_elements
+  for tile in dash.dashboard_elements:
+    assert tile.title
+    if tile.title.lower() == title:
+      found = tile
+  if not tile:
+    print(f"tile {title} of dashboard {dash.title} was not found")
+  return tile
+
+def download_tile(tile: models.DashboardElement, format:str = 'png'):
+  """Download the dashboard tile in the specified format"""
+  if not tile.query_id:
+    print(f"tile {tile.title} has no query_id")
+    return None
+  print(tile.query_id)
+  task = sdk.create_query_render_task(
+      query_id=tile.query_id, result_format=format, width=500, height=500)
+
+  if (not task or not task.id):
+    print(f"Could not create a render task for {tile.title}")
+    return None
+
+  # poll the render task until it completes
+  elapsed = 0.0
+  delay = 0.5 # wait .5 seconds
+  while True:
+    poll = sdk.render_task(task.id)
+    if poll.status == 'failure':
+      print(poll)
+      print(f"Render failed for {tile.title}")
+      return None
+    elif poll.status == 'success':
+      break
+
+    time.sleep(delay)
+    elapsed += delay
+    print(f"{elapsed} seconds elapsed")
```

Comment:
at some point it will probably be useful to create a `trace` or `log` function so we can enable/disable status output in one place